tools/ci_build/github/azure-pipelines/win-ci-pipeline.yml

```diff
@@ -4,39 +4,104 @@ parameters:
   type: boolean
   default: true
 
-jobs:
-- template: templates/win-cpu-ci.yml
-  parameters:
-    BuildConfig: 'Debug'
-    UseOmp: '--use_openmp'
-    EnvSetupScript: setup_env.bat
-    buildArch: x64
-    additionalBuildFlags: --use_dnnl --build_java --build_nodejs
-    msbuildPlatform: x64
-    isX86: false
-    job_name_suffix: x64_debug
-    RunOnnxRuntimeTests: ${{ parameters.RunOnnxRuntimeTests }}
+stages:
+- stage: x64_debug
+  dependsOn: []
+  jobs:
+    - template: templates/win-cpu-ci.yml
+      parameters:
+        BuildConfig: 'Debug'
+        UseOmp: '--use_openmp'
+        EnvSetupScript: setup_env.bat
+        buildArch: x64
+        additionalBuildFlags: --use_dnnl --build_java --build_nodejs --build_wheel
+        msbuildPlatform: x64
+        isX86: false
+        job_name_suffix: x64_debug
+        RunOnnxRuntimeTests: ${{ parameters.RunOnnxRuntimeTests }}
+        RunStaticCodeAnalysis: false
 
-- template: templates/win-cpu-ci.yml
-  parameters:
-    BuildConfig: 'RelWithDebInfo'
-    UseOmp: ''
-    EnvSetupScript: setup_env.bat
-    buildArch: x64
-    additionalBuildFlags: --use_dnnl --build_java --build_nodejs
-    msbuildPlatform: x64
-    isX86: false
-    job_name_suffix: x64_release
-    RunOnnxRuntimeTests: ${{ parameters.RunOnnxRuntimeTests }}
+- stage: x64_release
+  dependsOn: []
+  jobs:
+    - template: templates/win-cpu-ci.yml
+      parameters:
+        BuildConfig: 'RelWithDebInfo'
+        UseOmp: ''
+        EnvSetupScript: setup_env.bat
+        buildArch: x64
+        # Compare to our Nuget packaging pipeline, this job has "--build_wheel --use_dnnl" but doesn't have "--enable_lto --disable_rtti --use_telemetry  --enable_wcos"
+        # Python bindings use typeid so I can't disable RTTI here. So we may need to split it to two pipelines
```

Comment:
>So we may need to split it to two pipelines [](http://example.com/codeflow?start=67&length=43)

what would the two pipelines be?

Reply:
One to mimic the python packaging pipeline, another to mimic the nuget one. 
Because the python one has “build_wheel” and the nuget one has “disable_rtti”. They can’t co-exist. 
If there is a problem than only happens when RTTI is disabled and WinML is not enabled too, our CI can not catch it. But I think the risk is small.